app/test/tests/eventHandlerTest.ts

```diff
@@ -1,50 +1,50 @@
 
 var MockExpressRequest = require("mock-express-request");
 var MockExpressResponse = require("mock-express-response");
 
 // BEGIN TESTS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-export async function runCreateTest(db: firestore.Firestore) {
-  let mockBody: CreateEventRequest = {
-    orgId: "stacy@gmail.com",
-    name: "barb-eve",
-    description: "eve is the best",
-    startDate: new Date(),
-    endDate: new Date(),
-    location: {
-      room: "stay home",
-      placeId: "123-456-789",
-      building: "stay home",
-    },
-    tags: ["#social distance", "#six feet apart"],
-    media: ""
-  };
-  let mockRequest = new MockExpressRequest({
-    method: "POST",
-    url: "/createEvent",
-    body: mockBody
-  });
-  let mockResponse = new MockExpressResponse();
+// export async function runCreateTest(db: firestore.Firestore) {
+//   let mockBody: CreateEventRequest = {
+//     orgId: "stacy@gmail.com",
+//     name: "barb-eve",
+//     description: "eve is the best",
+//     startDate: new Date(),
+//     endDate: new Date(),
+//     location: {
+//       room: "stay home",
+//       placeId: "123-456-789",
+//       building: "stay home",
+//     },
+//     tags: ["#social distance", "#six feet apart"],
+//     media: ""
+//   };
+//   let mockRequest = new MockExpressRequest({
+//     method: "POST",
+//     url: "/createEvent",
+//     body: mockBody
+//   });
+//   let mockResponse = new MockExpressResponse();
 
-  // Simulate the app's sending of handler output to response
-  mockResponse.json(
-    await eventHandler.createEvent(db, mockRequest, mockResponse)
-  );
+//   // Simulate the app's sending of handler output to response
+//   mockResponse.json(
+//     await eventHandler.createEvent(db, mockRequest, mockResponse)
+//   );
 
-  // Check actual Firebase data
-  let mockResult = mockResponse._getJSON();
-  let eventResult = await db.collection("events").doc(mockResult.eventId).get()
-    .then(doc => {
-      describe("Expect event to exist").expect(doc.exists).toBe.equalTo(true);
-      return doc.data();
-    });
-    describe("Event description should be correct")
-    .expect(eventResult?.description).toBe.equalTo("eve is the best");
-    describe("Event has an organizer").expect(eventResult?.organizer).is.defined();
-}
+//   // Check actual Firebase data
+//   let mockResult = mockResponse._getJSON();
+//   let eventResult = await db.collection("events").doc(mockResult.eventId).get()
+//     .then(doc => {
+//       describe("Expect event to exist").expect(doc.exists).toBe.equalTo(true);
+//       return doc.data();
+//     });
+//     describe("Event description should be correct")
+//     .expect(eventResult?.description).toBe.equalTo("eve is the best");
+//     describe("Event has an organizer").expect(eventResult?.organizer).is.defined();
+// }
```

Comment:
What's wrong with this test?

Reply:
I think I uncommented it in another commit -- didnt mean to comment it out